lib/llm/providers/gemini.rb

```diff
@@ -18,19 +18,19 @@ def initialize(secret)
     end
 
     def embed(input, **params)
-      path = [PATH, "text-embedding-004"].join("/")
```

Comment:
@antaz Did we forget to delete the _assignment of the constant_ from this file ?

Reply:
@0x1eef We certainly did! Thank you